service/routes.py

```diff
@@ -20,12 +20,12 @@
 This service implements a REST API that allows you to manage Orders for a financial service.
 """
 
-from flask import jsonify
+from flask import jsonify, request
```

Comment:
Change to ```from flask import jsonify``` to comply with pylint standards, as that code is imported but not used.